JavaScript/README.md

```diff
@@ -16,6 +16,7 @@
 - [Boyer Moore's Majority Vote Algorithm](Algorithms/BoyerMooreAlgorithm.js)
 - [Armstrong Number](Algorithms/Armstrong.js)
 - [Longest Increaing Subsequnce](Algorithms/LongestIncreasingSubsequenceDP.js)
+- [Maximum Sum of Subarray](Algorithms/kadane's_algo.js)
```

Comment:
rename it ` [Maximum Sum of Subarray]` to `Kadane's Algorithm`

Reply:
done please check it